cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsRequestExecutor.java

```diff
@@ -203,7 +203,9 @@ private Object computeQueueLength(MetricsRequest metricsRequest) {
       if (queueName.isStream()) {
         streamNames.add(queueName.getSimpleName());
       } else if (queueName.isQueue()) {
-        String context = String.format("%s.f.%s.%s",
+        // TODO: After CDAP-1167 is fixed, namespace should be the first component of the queueName.
```

Comment:
I see that you have a PR for CDAP-1167. So will you update this PR after that is merged? Or will there be a third PR after these two gets merged?